tests/manage/pv_services/pvc_resize/test_node_restart_during_pvc_expansion.py

```diff
@@ -92,11 +92,49 @@ def test_worker_node_restart_during_pvc_expansion(self, nodes):
 
         # Verify pvc expansion status
         for pvc_obj in self.pvcs:
-            assert (
-                pvc_obj.expand_proc.result()
-            ), f"Expansion failed for PVC {pvc_obj.name}"
+            assert pvc_obj.expand_proc.result(), (
+                f"Expansion failed for PVC {pvc_obj.name}\nDescribe output "
+                f"of PVC and PV:\n{pvc_obj.describe()}\n"
+                f"{pvc_obj.backed_pv_obj.describe()}"
+            )
         log.info("PVC expansion was successful on all PVCs")
 
+        log.info("Verifying new size on pods.")
+        for pod_obj in self.pods:
+            if pod_obj.pvc.volume_mode == "Block":
+                log.info(
+                    f"Skipping check on pod {pod_obj.name} as volume mode is Block."
+                )
+                continue
+
+            # Wait for 240 seconds to reflect the change on pod
+            log.info(f"Checking pod {pod_obj.name} to verify the change.")
+            for df_out in TimeoutSampler(
+                240, 3, pod_obj.exec_cmd_on_pod, command="df -kh"
```

Comment:
worth moving to utils so that others can utilize

Reply:
Good to add a helper function which can fetch the total size, used size or used percentage as needed.